server/monitor/src/main/java/org/apache/accumulo/monitor/EmbeddedWebServer.java

```diff
@@ -118,19 +98,17 @@ public void start() {
     }
   }
 
-  public void stop() {
+  private void stop() {
```

Comment:
This is part of the work @ctubbsii did, so I am not sure what the reason behind this is.